### ISSUE TITLE
[QUESTION] Can @tsed/react-formio be used without Ts.ED framework?

### ISSUE THREAD

**Issue by mm-dsibinski** [tsedio/tsed-formio#117]
Hi!
Can `@tsed/react-formio` be considered a better alternative to `@formio/react` package?
In other words: is `tsed-formio` somehow tied to [Ts.ED](https://tsed.io/) framework?

I'd like to use it instead of `@formio/react` in a typical React project.

**Comment by Romakita** [comment]
Hello @mm-dsibinski 
It’s totally independant of the tsed framework. It’s effectivly annalternative of react-formio ;)

**Comment by mm-dsibinski** [comment]
Thanks for answering @Romakita 🙂
I checked the package, and it seems you're doing a great job here! 💪

However, I see that the React API is a bit different from original FormIO's package. Which is not a disadvantage of course, but I noticed that `FormBuilder` component from `@tsed/react-formio` expects to separately provide `components` and `display`, while the original one accepts the whole `form` object.  It's a bit problematic for me to use now, because I also provide some custom objects there (except `components` and `display`). Unfortunately, I had to do it to "hack" around some of FormIO caveats (🤪) and it's not easy to be refactored now.
